tables/utils.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+
+
+def column_index_to_excel_letters(col_idx: int) -> str:
+    """Converts an integer column index (0-indexed) to Excel column letters.
```

Comment:
this may seem a bit picky, but if passed a negative integer as col_idx (say last position in Python style indexing) it will return "", but should it just error? I.e. only accept positive integers?

Reply:
in a similar vein, the max number of columns in Excel in 16,384, so could stipulate the input index must be between 0 and 16,384 otherwise ValueError?

Reply:
good shout - have added some input validation this function now